crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -91,4 +89,15 @@ message SealAndExtendChainRequest {
   string params = 5;
 }
 
-message SealAndExtendChainResponse {}
\ No newline at end of file
+message SealAndExtendChainResponse {}
+
+message FindTailRequest { uint32 log_id = 1; }
+
+message FindTailResponse {
```

Comment:
A potential improvement is to also return the loglet params, for that we'll need to add a getter in the Loglet trait, this can be in a follow up.